client/js/mousehandling.js

```diff
@@ -36,7 +36,7 @@ async function inputHandler(name, e) {
     document.activeElement.blur();
   }
   let target = e.target;
-  while(target && (!target.id || !widgets.has(target.id)))
+  while(target && (!target.id || target.id.slice(0,2) != 'w_' || !widgets.has(unescapeID(target.id.slice(2)))))
```

Comment:
Shouldn't actually nee to check widgets has here any more since any element with id beginning with `w_` should represent a widget.

Reply:
True. But it feels like it should stay there just in case. :rofl: